librocksdb_sys/src/lib.rs

```diff
@@ -263,6 +263,9 @@ extern "C" {
     pub fn crocksdb_options_set_stats_dump_period_sec(options: *mut DBOptions, v: usize);
     pub fn crocksdb_options_set_num_levels(options: *mut DBOptions, v: c_int);
     pub fn crocksdb_options_set_db_log_dir(options: *mut DBOptions, path: *const c_char);
+    pub fn crocksdb_options_set_wal_dir(options: *mut DBOptions, path: *const c_char);
+    pub fn crocksdb_options_set_WAL_ttl_seconds(options: *mut DBOptions, ttl: u64);
```

Comment:
It is uppercase to match the function name in rocksdb C library, as for crocksdb_writeoptions_disable_WAL function.
I can lowercase if you prefer, just confirm.